examples/kitchen-sink/apps/blog/remix.config.js

```diff
@@ -1,9 +1,15 @@
 /**
- * @type {import('@remix-run/dev/config').AppConfig}
+ * @type {import('@remix-run/dev').AppConfig}
  */
 module.exports = {
-  appDirectory: "app",
```

Comment:
Need to specify correct outputs or change in turbo.json

Reply:
Updated!